ui/lib/kv/addon/routes/secret/details/index.js

```diff
@@ -5,17 +5,35 @@
 
 import Route from '@ember/routing/route';
 import { breadcrumbsForSecret } from 'kv/utils/kv-breadcrumbs';
+import { inject as service } from '@ember/service';
 
 export default class KvSecretDetailsIndexRoute extends Route {
-  setupController(controller, resolvedModel) {
+  @service store;
+
+  // polled by controller and called once when route initializes
+  async fetchSyncStatus(model) {
```

Comment:
Should follow-up with design if this should poll...since it's a status it made sense?